src/NUnitFramework/tests/Attributes/OSPlatformAttributeTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
+#if NET5_0
```

Comment:
Instead of explicitly targeting NET5 by name, can we invert the check to be "NOT {UNSUPPORTED}". That way these tests will be included in other future build targets, NET6+.

For example something like (untested):
```csharp
#if !(NET45 | NETCOREAPP)
// net 5 code here
#endif
```

Reply:
I would have liked to use `#if NET5_0_OR_GREATER` but it wasn't available in the SDK used by NUnit. Is there a reason we are not using the latest minor version 5.0.400?

Reply:
That's a good question! I'm not sure

Reply:
Usually, we have only bumped these when necessary to make it possible for people to contribute even if they are not using the latest SDKs. And often we only updated if some CI broke.

But I cannot see a large problem with bumping to 5.0.400 (except that I need to update my VS installation). @rprouse do you have any oppinon on this?

Reply:
I am fine to bump the minor version, we should be running the latest bug and security fixes.

Reply:
Actually the bump by @mikkelbu to 5.0.202 added NET5_0_OR_GREATER already. Although I couldn't find it in any release notes.
Rebased the branch and updated the code to use '#if NET5_0_OR_GREATER' to fix @stevenaw concern.

Reply:
Thanks @manfred-brands , that looks great! 👍